tooling/components/io.siddhi.distribution.editor.core/src/main/resources/web/editor/commons/js/dialog/extension-install-dialog.js

```diff
@@ -2,39 +2,34 @@
  * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org)  Apache License, Version 2.0
  * http://www.apache.org/licenses/LICENSE-2.0
  */
-define(['require', 'lodash', 'jquery', 'log', 'backbone'],
-    function (require, _, $, log, Backbone) {
-        var ExtensionInstallDialog = Backbone.View.extend(
-            /** @lends ExtensionInstallDialog.prototype */
-            {
-                /**
-                 * @augments Backbone.View
-                 * @constructs
-                 * @class ExtensionInstallDialogExtensionInstallDialog
-                 * @param {Object} config configuration options for ExtensionInstallDialog
-                 */
-                initialize: function (options) {
-                    // this.app = options;
-                    this.dialog_containers = $(_.get(options.config.dialog, 'container'));
-                    this.notification_container = _.get(options.config.tab_controller.tabs.tab.das_editor.notifications,
-                        'container');
-                },
+define(['require', 'lodash', 'jquery'],
+    function (require, _, $) {
+        /** @lends ExtensionInstallDialog.prototype */
+        var ExtensionInstallDialog = function ()
+             {
+            /**
+             * @constructs
+             * @class ExtensionInstallDialogExtensionInstallDialog
+             * @param {Object} config configuration options for ExtensionInstallDialog
+             */
+            this.initialize = function (options) {
+                this.dialog_containers = $(_.get(options.config.dialog, 'container'));
+                this.extensionList = getExtensionDetail();
```

Comment:
change method name to getExtensionDetails

Reply:
9ed07404e8a403916fc57b790dd8f95a62784477 this commit  will be resolving above request.